ports/espressif/boards/maker_badge/mpconfigboard.mk

```diff
@@ -0,0 +1,29 @@
+USB_VID = 0x239A
+USB_PID = 0x2030
+USB_PRODUCT = "Maker badge"
+USB_MANUFACTURER = "Czech maker"
+
+IDF_TARGET = esp32s2
+
+INTERNAL_FLASH_FILESYSTEM = 1
+LONGINT_IMPL = MPZ
+
+# The default queue depth of 16 overflows on release builds,
+# so increase it to 32.
+CFLAGS += -DCFG_TUD_TASK_QUEUE_SZ=32
```

Comment:
These lines are not needed, because these are defaults on espressif.  Remove them please.

Reply:
Thanks, I removed those lines 🙂